### DISCUSSION TITLE
IR spectra from deep WFC

### DISCUSSION THREAD

**Discussion by AxelTG** [deepmodeling/deepmd-kit#723] [Q&A]
Hi, I would like to train a model from which IR spectra can be computed, such as in [10.1103/PhysRevB.102.041121](https://arxiv.org/abs/1906.11434). I've tried to use the provided input file [wannier.json](https://github.com/deepmodeling/deepmd-kit/blob/master/examples/water/train/wannier.json) but this error occurred: 'get invalid choice wfc for flag type.' Is this option not supported anymore? 
I've also tried to train [polar.json](https://github.com/deepmodeling/deepmd-kit/blob/master/examples/water/train/polar.json) but another error occurred: '../data-shuffled/set.000/polarizability.npy not found!'. What is the structure of the data folder (and the units) for such a training? What is the difference between the two fitting net types (wfc and polar)?
Thank you.

**Comment by YWolfeee** [comment]
Hi, `wfc` is no longer supported in `devel` branch. You can use wannier center to calculate wannier centroid, and use `dipole` mode to fit wannier centroid.

The structure of data folder can be founded in `examples/water_tensor/dipole` in `devel` branch. If your label is global dipole, then it should be stored in `dipole.npy`. Otherwise, it should be stored in `atomic_dipole.npy`. You can use the script `$deepmd_source_dir/data/raw/raw_to_set.sh` to convert the prepared raw files (`dipole.raw`, `atomic_dipole.raw`) to data sets (`dipole.npy`, `atomic_dipole.npy`).
